integration/tests/next-build.test.ts

```diff
@@ -89,6 +89,15 @@ export default function RootLayout({ children }: { children: React.ReactNode })
 
     expect(notFoundPageLine).toContain(staticIndicator);
   });
+
+  /**
+   * Sometimes utilities from `/server` may use Node APIs even if `clerkMiddleware` does not consumes them.
+   * This happends because of code for node runtime and edge runtime is bundled together in the `/server/index.ts` barrel file.
+   * This test ensures that developers will not end up with warnings on `next build`.
```

Comment:
```diff
- * Sometimes utilities from `/server` may use Node APIs even if `clerkMiddleware` does not consumes them.
+ * Sometimes utilities from `/server` may use Node APIs even if `clerkMiddleware` does not consume them.
- * This happends because of code for node runtime and edge runtime is bundled together in the `/server/index.ts` barrel file.
+ * This happens when code for node runtime and edge runtime is bundled together in the `/server/index.ts` barrel file.
* This test ensures that developers will not end up with warnings on `next build`.
```